evennia/contrib/worldloader/example_empty_world/worlddata/models.py

```diff
@@ -0,0 +1,121 @@
+from django.db import models
+
+
+#------------------------------------------------------------
+#
+# store all rooms
+#
+#------------------------------------------------------------
+class world_rooms(models.Model):
```

Comment:
These tables store game world data. Currently CSV file is the only data source, but in the future, other files could be data source too, or even modify data online. All these data sources will be imported into these tables, and use these tables to create the game world.